packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -1022,17 +1022,11 @@ const runGenerator = async (
         return acc
     }, {})
 
+    // TODO: consider "config/default.js" (cosmiconfig) instead?
```

Comment:
I think for now.. lets stick with placing the configuration in the package.json. That fits better with the notion that template-minimal is a minimal template. For advanced config usages the developer my seek to look at documentation to change where the project houses its config.